src/ctap/mod.rs

```diff
@@ -211,37 +211,43 @@ fn truncate_to_char_boundary(s: &str, mut max: usize) -> &str {
 
 /// Parses the metadata of an upgrade, and checks its correctness.
 ///
-/// The metadata consists of:
+/// The metadata is a page starting with:
```

Comment:
By the way, this is specific to Nordic. A page is not necessarily 4kB. It makes sense for the bootloader which is Nordic-specific, but ctap-code should ideally be abstract over a page size. But I guess it's fine to keep it like that or rename "page" to "slice" or something.

Reply:
Ah yeah, no problem. This is moving to Env in 2 PRs.